server/planning/planning.py

```diff
@@ -47,11 +48,6 @@ def on_create(self, docs):
         for doc in docs:
             doc['guid'] = generate_guid(type=GUID_NEWSML)
             set_original_creator(doc)
-            # remove event expiry if it is linked to the planning
```

Comment:
Event, Agenda and Planning items should only be removed from the system when they're marked as spiked and purged from the system, based on a configurable number of minutes from on the **PLANNING_EXPIRY_MINUTES** system setting. Please see [SDESK-1224](https://dev.sourcefabric.org/browse/SDESK-1224) for futher information

There is logic in place to ensure that if an Event is spiked that all associated Planning items are marked as spiked as well, which the end user is informed about in the SpikeEvent modal component.